ruleset/rules/0998-aws-security-hub-rules.xml

```diff
@@ -0,0 +1,491 @@
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - RDS snapshot is public.</description>
+    <group>pci_dss_1.2.1,pci_dss_1.3.1,pci_dss_1.3.4,pci_dss_1.3.6,pci_dss_7.2.1,nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99853" level="12">
+    <if_sid>99827</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^RDS.2$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Amazon RDS instance is publicly accessible.</description>
+    <group>pci_dss_1.2.1,pci_dss_1.3.1,pci_dss_1.3.4,pci_dss_1.3.6,pci_dss_7.2.1,nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99854" level="12">
+    <if_sid>99828</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^Redshift.1$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub -  Amazon Redshift cluster is publicly accessible.</description>
+    <group>pci_dss_1.2.1,pci_dss_1.3.1,pci_dss_1.3.2,pci_dss_1.3.4,pci_dss_1.3.6,nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99855" level="12">
+    <if_sid>99829</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^S3.2$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Amazon S3 general purpose bucket permits public read access.</description>
+    <group>pci_dss_1.2.1,pci_dss_1.3.1,pci_dss_1.3.2,pci_dss_1.3.6,pci_dss_7.2.1,nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99856" level="12">
+    <if_sid>99829</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^S3.3$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Amazon S3 general purpose bucket permits public write access.</description>
+    <group>pci_dss_1.2.1,pci_dss_1.3.1,pci_dss_1.3.2,pci_dss_1.3.4,pci_dss_1.3.6,pci_dss_7.2.1,nist_800_53_AC.3,nist_800_53_AC.4,nist_800_53_AC.6,nist_800_53_AC.21,nist_800_53_SC.7,</group>
+  </rule>
+
+  <rule id="99857" level="12">
+    <if_sid>99829</if_sid>
+    <field name="aws.finding.Compliance.SecurityControlId">^S3.19$</field>
+    <field name="aws.finding.Compliance.Status">FAILED</field>
+    <field name="aws.finding.RecordState">ACTIVE</field>
+    <description>AWS Security Hub - Amazon S3 access point block public access settings aren't enabled.</description>
```

Comment:
Change to "are not".

Reply:
This has been resolved https://github.com/wazuh/wazuh/pull/23234/commits/1d4bfae8c2701e7cc0cda7b3f8df0965bdb225f8.